networking/ovn_kubernetes_network_provider/configuring-gateway-mode.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-gateway-mode"]
+= Configuring gateway mode
+include::_attributes/common-attributes.adoc[]
+:context: configuring-gateway-mode
+
+toc::[]
+
+As a cluster administrator you can configure the `gatewayConfig` object to manage how external traffic leaves the cluster. You do so by setting the `routingViaHost` spec to `true` for local mode or `false` for local mode.
```

Comment:
In the second sentence, it states that both options are for local mode, is this correct?